internal/cmdreporeg/command_test.go

```diff
@@ -13,3 +13,215 @@
 // limitations under the License.
 
 package cmdreporeg
+
+import (
+	"encoding/json"
+	"flag"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/GoogleContainerTools/kpt/internal/printer/fake"
+	"github.com/google/go-cmp/cmp"
+	"gopkg.in/yaml.v3"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var (
+	update = flag.Bool("update", false, "update golden files")
```

Comment:
Nit: I personally have used env vars for this ... I've created a library that I'm pretty happy with, but unfortunately it is off on an experimental branch: https://github.com/justinsb/kpt/blob/main_integration/porch/functions/pkg/testhelpers/golden.go#L53  Not a blocker because we can move across fairly easily later!